content/guides/installer-guides/containerized/_index.md

```diff
@@ -0,0 +1,200 @@
+[osdsapiserver]
+api_endpoint = 0.0.0.0:50040
+auth_strategy = keystone
+# If https is enabled, the default value of cert file
+# is /opt/opensds-security/opensds/opensds-cert.pem,
+# and key file is /opt/opensds-security/opensds/opensds-key.pem
+https_enabled = False
+beego_https_cert_file =
+beego_https_key_file =
+# Encryption and decryption tool. Default value is aes.
+password_decrypt_tool = aes
+
+[keystone_authtoken]
+memcached_servers = 127.0.0.1:11211
+signing_dir = /var/cache/opensds
+cafile = /opt/stack/data/ca-bundle.pem
+auth_uri = http://127.0.0.1/identity
+project_domain_name = Default
+project_name = service
+user_domain_name = Default
+password = opensds@123
+# Whether to encrypt the password. If enabled, the value of the password must be ciphertext.
+enable_encrypted = False
+# Encryption and decryption tool. Default value is aes. The decryption tool can only decrypt the corresponding ciphertext.
+pwd_encrypter = aes
+username = opensds
+auth_url = http://127.0.0.1/identity
+auth_type = password
+
+[osdslet]
+api_endpoint = 127.0.0.1:50049
+
+[osdsdock]
+api_endpoint = 127.0.0.1:50050
+# Choose the type of dock resource, only support 'provisioner' and 'attacher'.
+dock_type = provisioner
+# Specify which backends should be enabled, sample,ceph,cinder,lvm and so on.
+enabled_backends = $BackendType
+
+[sample]
+name = sample
+description = Sample Test
+driver_name = sample
+
+[lvm]
+name = lvm
+description = LVM Test
+driver_name = lvm
+config_path = /etc/opensds/driver/lvm.yaml
+host_based_replication_driver = DRBD
+
+[database]
+endpoint = 127.0.0.1:2379,127.0.0.1:2380
+driver = etcd
+OPENSDS_GLOABL_CONFIG_DOC
+```
+
+If you choose `lvm` as backend, you need to make sure physical volume and volume group existed. Besides, you need to configure lvm driver.
+```
+sudo pvdisplay # Check if physical volume existed
+sudo vgdisplay # Check if volume group existed
+
+mkdir -p /etc/opensds/driver && sudo cat > /etc/opensds/driver/lvm.yaml <<OPENSDS_DRIVER_CONFIG_DOC
+tgtBindIp: 127.0.0.1
+tgtConfDir: /etc/tgt/conf.d
+pool:
+  {{ volume_group_name }}:
+    storageType: block
+    availabilityZone: default
+    extras:
+      dataStorage:
+        provisioningPolicy: Thin
+        isSpaceEfficient: false
+      ioConnectivity:
+        accessProtocol: iscsi
+        maxIOPS: 7000000
+        maxBWS: 600
+      advanced:
+        diskType: SSD
+        latency: 5ms
+OPENSDS_DRIVER_CONFIG_DOC
+```
+
+### SODA Service Installation
+If you are a lazy one, just like me, you probably want to do this:(`docker-compose` required)
+```
+wget https://raw.githubusercontent.com/opensds/opensds/development/docker-compose.yml
+
+docker-compose up -d
+```
+
+Or you can do this:
+```
+docker run -d --net=host -v /usr/share/ca-certificates/:/etc/ssl/certs quay.io/coreos/etcd:latest
+
+docker run -d --net=host --privileged=true opensdsio/opensds-authchecker:latest
+
+docker run -d --net=host -v /etc/opensds:/etc/opensds opensdsio/opensds-apiserver:latest
```

Comment:
These executable names may not be valid for recent releases. Can we add some notes  that these steps are to be validated.